SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -472,3 +337,176 @@ void USpatialWorkerConnection::OnConnectionFailure()
 		GEngine->BroadcastNetworkFailure(GameInstance->GetWorld(), GetSpatialNetDriverChecked(), ENetworkFailure::FromDisconnectOpStatusCode(ConnectionStatusCode), *ErrorMessage);
 	}
 }
+
+bool USpatialWorkerConnection::Init()
+{
+	OpsUpdateInterval = 1.0f / GetDefault<USpatialGDKSettings>()->OpsUpdateRate;
```

Comment:
Why can't we make these settings memebers constexpr values somewhere?

Reply:
Potentially due to how settings in Unreal work.

Reply:
Also they can't really be constexpr since they can change during runtime by the user.

Reply:
Fair enough.